documents/papers/chep2019/modern-stack.tex

```diff
@@ -65,16 +65,16 @@
 \section{Introduction}
 \label{intro}
 
-Software plays a critical part in the lifecycle of High-Energy Physics (HEP)
+Software plays a critical role in the lifecycle of High-Energy Physics (HEP)
 experiments. From the high-level triggers of the experiments, through the
 reconstruction chain, to the identification of analysis objects, and finally to
 the final physics analysis, software is ubiquitous. In parallel, the simulation
 chain, generating physics events and simulating their passage through the
-detector compliments this data driven chain and is entirely done by software.
+detector, compliments this data driven chain and is entirely done by software.
 
 As HEP software has become more sophisticated so the dependencies on libraries
 and supporting software components has become more important. This leads to the need to
-build software as a deep and wide stack, which, for a modern HEP experiment, may
+build software as a deep and wide (Define these?) stack, which, for a modern HEP experiment, may
```

Comment:
Yes, it's worth adding detail on what we mean here. I'll do that in the next commit.